poll/poll.py

```diff
@@ -96,6 +100,18 @@ class PollBase(XBlock, ResourceMixin, PublishEventMixin):
         default=0, help=_("Number of times the user has sent a submission."), scope=Scope.user_state
     )
     feedback = String(default='', help=_("Text to display after the user votes."))
+    active_export_task_id = String(
+        # The UUID of the celery AsyncResult for the most recent export,
+        # IF we are sill waiting for it to finish
+        default="",
+        scope=Scope.user_state_summary,
+    )
+    last_export_result = Dict(
+        # The info dict returned by the most recent successful export.
+        # If the export failed, it will have an "error" key set.
+        default=None,
+        scope=Scope.user_state_summary,
+    )
```

Comment:
Can you please move these fields and all the new methods into a mixin class, to keep the export-related code separate from the rest of the implementation?